samples/bluetooth/fast_pair/input_device/README.rst

```diff
@@ -12,9 +12,11 @@ This sample demonstrates :ref:`how to use Google Fast Pair with the nRF Connect
 Google Fast Pair Service (GFPS) is a standard for pairing Bluetooth® and Bluetooth LE devices with as little user interaction required as possible.
 Google also provides additional features built upon the Fast Pair standard.
 For detailed information about supported functionalities, see the official `Fast Pair`_ documentation.
+The software maturity level for the input device use case is listed in the :ref:`software_maturity_fast_pair_use_case` table.
```

Comment:
This is still a note in the Locator Tag sample? Should we align so that samples look uniformly? Why was the SML information removed from the note information?

Reply:
Aligned Locator Tag note. I tried to avoid having 2 notes one after another here